react/src/components/atoms/buttons/ButtonAlert/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.css';
+import Icon from '../../icons/Icon';
+
```

Comment:
We should make sure that the onClick prop passed is a function, so that prod builds won't die. Currently we have a package that helps for checking types called `is`:
```suggestion
import is from 'is';

```

Reply:
Assuming this suggestion is accepted, we should change the return for Button alert so that it can define an function that returns content, and conditionally add onClick if it's a function:
```
const ButtonAlert = ({
  text, onClick, hideText, showText
}) => {
  const buttonProps = {
    className: 'ma__button-alert',
  };
  if (is.fn(onClick)) {
    buttonProps.onClick = onClick;
  }
  return(<button {...buttonProps}> ....rest of the code here);
}
```